_extras/guide.md

```diff
@@ -245,13 +245,20 @@ stack_selection.columns = stack_selection.columns.droplevel()
 
 ### Challenge - Changing Types
 
+* Try converting the column `plot_id` to floats using `surveys_df.plot_id.astype("float")`. Next try converting `weight` to an integer. What goes wrong here? What is Pandas telling you?
+
 Pandas cannot convert types from float to int if the column contains NaN values.
 
 ### Challenge - Counting
+
+* Count the number of missing values per column. Hint: The method .count() gives you the number of non-NA observations per column. Try looking to the .isnull() method.
```

Comment:
use backticks around .count() and .isnull() so that they appear as `.count()` and `.isnull()`

Reply:
```suggestion
* Count the number of missing values per column. Hint: The method `.count()` gives you the number of non-NA observations per column. Try looking to the `.isnull()` method.
```

If you just want to accept this change, @NPalopoli, it contains the correct change.